cudax/include/cuda/experimental/__cuco/hash_functions.cuh

```diff
@@ -30,7 +31,14 @@ namespace cuda::experimental::cuco
 
 enum class HashStrategy
```

Comment:
Can we use snake-case for the public API as it aligns better with with the naming conventions in STL/libcu++ and Thrust?